connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerGroupMember.java

```diff
@@ -151,10 +151,23 @@ public void stop() {
         stop(false);
     }
 
+    /**
+     * Ensure that the connection to the broker coordinator is up and that the worker is an
+     * active member of the group.
+     */
     public void ensureActive() {
         coordinator.poll(0);
     }
 
+    /**
+     * A lightweight wrapper call that answers whether there's an active connection to the broker
+     * coordinator.
+     * @return true if there is no active connection to a broker coordinator
+     */
+    public boolean coordinatorUnknown() {
```

Comment:
Where is this function used? It appears that there was already a `coordinatorUnknown()` function available in the WorkerCoordinator.

Reply:
Good catch. Remainder from an earlier attempt, when I thought I needed to expose this to the `DistributedHerder`. Indeed unused.